addons/website/static/src/js/menu/seo.js

```diff
@@ -613,10 +614,25 @@ var SeoConfigurator = Dialog.extend({
             self.renderPreview();
         });
     },
-    descriptionChanged: function () {
+    descriptionChanged: function (ev) {
         var self = this;
         _.defer(function () {
-            var description = self.$('textarea[name=seo_page_description]').val();
+            var $textarea = $(ev.target);
```

Comment:
Can you retrieve the textarea without using ev (website_meta_description)? 
And then call _onMetaDescriptionChanged on start event of the widget ?
That way the warning will be shown if there is a not required size description when loading the widget

Reply:
Ok